README.md

```diff
@@ -206,16 +206,21 @@ It's possible to run multiple same services, e.g. two `tezos-node`s that run dif
 networks.
 
 `tezos-node` packages provide three services out of the box:
```

Comment:
Well these are only two services, perhaps this is (and already was) outdated?

Unless you are also counting `tezos-node-custom` here as well?

Reply:
> provide multiple services
:smart: